### ISSUE TITLE
Add a figure/infographic listing the ten tips?

### ISSUE THREAD

**Issue by Benjamin-Lee** [Benjamin-Lee/deep-rules#261]
I was just looking online and saw someone tweeted a like to a Ten Simple Rules article which featured a figure showing the ten rules. On Twitter, it looked like this:

![image](https://user-images.githubusercontent.com/13618205/95688479-6e854000-0bd8-11eb-868f-4403af78b4bb.png)

Does anyone think this is a good idea for outreach?

**Comment by cgreene** [comment]
Yes - it also makes us keep the rules simple! 😁

**Comment by Benjamin-Lee** [comment]
Ok we can work on this after we freeze the content and start soliciting final approvals.

**Comment by rasbt** [comment]
Yeah, I think it would be nice to have a small infographic for the 10 tips like the one above (or the one below; in nice of course)

![Screen Shot 2020-10-12 at 9 10 56 PM](https://user-images.githubusercontent.com/5618407/95807334-868bbb00-0ccf-11eb-9521-985a8ec92a65.png)

But I'd say we should wait until any sharing/advertising until after the peer-reviews and potential acceptance for publication.

**Comment by Benjamin-Lee** [comment]
@rasbt what are your thoughts on including the infographic as fig 1 in the paper? That's what the paper I saw on twitter did (and why that was the thumbnail). As it stands, right now our thumbnail is the overfitting figure.

**Comment by rasbt** [comment]
Oh this was in the paper itself? I see. Hm, it's basically just the section titles, right? To be honest, having a figure listing the section titles sounds a bit overkill since the figure doesn't really add anything to the paper then. We could maybe have it as a summary in the beginning and end if it has some additional keyword phrases that summarize each section.

**Comment by Benjamin-Lee** [comment]
Yeah they put it in the introduction: https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008210

I haven't seen that done before which is why I raised it in an issue.

**Comment by rasbt** [comment]
Hm, I think for our paper this would be a bit too much -- we already have figures, so there is no need to create figures for the sake of having figures in the paper. However, we should probably make a flowchart figure for sharing it later on if/when it is accepted & published

**Comment by chevrm** [comment]
This one is a pretty nice aesthetic
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007809

**Comment by rasbt** [comment]
I agree. This one, with its short summaries, looks nice & useful to me

**Comment by fmaguire** [comment]
I've thrown together a rough draft of this infographic (text definitely needs tweaked and some of the icons probably miss the mark).  There is an SVG version in my fork (and current open PR) for modification/direct edits.

![tips_overview](https://user-images.githubusercontent.com/1698629/102679201-53105580-4184-11eb-8ccd-636db7659fa1.png)

**Comment by rasbt** [comment]
I really like it. The icons are great, I think. When I was reading through it, I could see why you picked them for the respective heading. It's a great infographic overall. Added some comments to the PR

**Comment by juancarmona** [comment]
@fmaguire this is excellent! A very minor typo: in No. 5, Architecture is misspelled. I just spoke with @Benjamin-Lee and perhaps placing this in the Intro would be great to guide the reader and also for when we post the manuscript in social media, as the primary figure is usually what's displayed for viewers.

**Comment by Benjamin-Lee** [comment]
This is fantastic! I love it. The icons convey the meaning perfectly and I definitely think we should make this our figure 1.

**Comment by fmaguire** [comment]
![tidy_shifted](https://user-images.githubusercontent.com/1698629/102797221-c503bd00-4385-11eb-9c31-51037933c9be.png)

Hey folks, I've integrated the comments here and on the PR, as well as done some tidying up of the figure (consistent use of style and colour, simplifying more complex icons where possible, keeping text in consistent places to make it look less messy).

To avoid doing lots of changes in the repo of a binary image (or big ole SVG), let me know if you there is anything else you want changed (and once finalised I'll update the PR accordingly).

**Comment by rasbt** [comment]
It looks great! No further suggestions from my side

**Comment by Benjamin-Lee** [comment]
A few other ideas. You are clearly a very strong designer (much better than me) so do as you see fit:

1. The interpretability tip icon is a little confusing, though the new color is an improvement. Is it supposed to represent a black box? If so, maybe it should be black?
2. Do we maybe want to put the title of the paper at the top like the example in https://github.com/Benjamin-Lee/deep-rules/issues/261#issue-718912200
3. You use [ ] around the data in tip 1. Maybe use that also in 4?

**Comment by fmaguire** [comment]
![tips_overview](https://user-images.githubusercontent.com/1698629/102959186-3beba380-44b5-11eb-9c75-cd8d7b1ce3ef.png)

Good idea on the title and making the data dangers icon consistent.

I tried for a bit to make the interpretability less ambiguous but couldn't really come up with much in a style that matched that wasn't way to complex for an icon. "Opening the box on your model's inner workings"/"Opening the blackbox" was the thinking.

Re: colour, black would be ideal but testing it makes it really stand out and visually inconsistent with the other icons (see the first draft above on the microscope for a similar example).  I've kinda tied my own hands with the palette I've used for the other icons so I'm stuck with the darkest option: purple!  

I've integrated what I can and updated the PR with the latest SVG and PNG.

**Comment by Benjamin-Lee** [comment]
> Re: colour, black would be ideal but testing it makes it really stand out and visually inconsistent with the other icons (see the first draft above on the microscope for a similar example). I've kinda tied my own hands with the palette I've used for the other icons so I'm stuck with the darkest option: purple!

This makes sense. Purple it is! You're also right that any other icon would be too complex.

Last question: Tip 7's points are purple with orange lines while 2 and 6 are mostly orange points with black lines. Do the colors represent something specific?

**Comment by agitter** [comment]
Excellent summary figure @fmaguire!

I like @juancarmona's suggestion to set this as the thumbnail for social media.  @Benjamin-Lee can do that from the repository settings.  I can also help set this image as the manuscript thumbnail for the Manubot catalog.  I don't want to create extra work, but a copy that only has the title text and uses a larger font size would probably look best for a thumbnail image.  This more detailed version could go in the manuscript.

**Comment by Benjamin-Lee** [comment]
@agitter I completely forgot about the repo thumbnail setting. I'll absolutely update it once we merge.

**Comment by fmaguire** [comment]
I lied, had a few minutes while something was running.



> Last question: Tip 7's points are purple with orange lines while 2 and 6 are mostly orange points with black lines. Do the colors represent something specific?

Good catch, I've swapped to orange and green for more consistency.

> Thumbnail

I've made a thumbnail version with larger rule titles only. As these titles are better than what I had originally, I also added them to the full figure/graphical abstract version.

![thumbnail_tips_overview](https://user-images.githubusercontent.com/1698629/103042865-5a9f7800-4551-11eb-9d67-b008630fbfbd.png)


![tips_overview](https://user-images.githubusercontent.com/1698629/103042871-5f642c00-4551-11eb-9d9b-b150a8240fc8.png)
